lib/linux_kernel/atomic64.h

```diff
@@ -83,7 +83,7 @@ m0_atomic64_sub_return(struct m0_atomic64 *a, int64_t delta)
 
 static inline bool m0_atomic64_inc_and_test(struct m0_atomic64 *a)
 {
-	return atomic64_inc_and_test(&a->a_value);
+	return (&a->a_value);
```

Comment:
If the ARM process does not support this, at least we need to increase the value here:
```
++ a->a_value;
return (a->a_value == 0);
```

Reply:
Fixed the missing function name  accidentally deleted against it  "m0_atomic64_inc_and_test(&a->a_value)